src/components/DisplayText.vue

```diff
@@ -11,20 +11,56 @@ img {
 </style>
 
 <script>
+import 'katex/dist/katex.min.css'
 import showdown from 'showdown'
+import showdownKatex from 'showdown-katex'
 
 export default {
     props: ['text'],
 
     data() {
         return {
-            converter: new showdown.Converter(),
+            converter: new showdown.Converter({
+                extensions: [
+                    showdownKatex({
+                        displayMode: true,
+                        throwOnError: false, // allows katex to fail silently
+                        errorColor: '#ff0000',
+                        delimiters: [
+                            { left: '$', right: '$' },
+                            {
+                                left: '~',
+                                right: '~',
+                                display: false,
+                                asciimath: true,
+                            },
+                        ],
+                    }),
+                ],
+            }),
         }
     },
 
     computed: {
         parsed() {
-            return this.converter.makeHtml(this.text)
+            let newText = this.text
+            // The regex expression is used to find
+            // substrings which are latex blocks and ignores \$
+            const latexRegex = /(?<!\\)\$.*?(?<!\\)\$/g
+            const matches = newText.matchAll(latexRegex)
+            for (const match of matches) {
+                const startIndex = match.index
+                const endIndex = match.index + match[0].length
+                let substring = newText.slice(startIndex, endIndex)
+                substring = substring.replaceAll('*', '\\*')
+                substring = substring.replaceAll('_', '\\_')
+
+                newText =
+                    newText.slice(0, startIndex) +
+                    substring +
+                    newText.slice(endIndex)
```

Comment:
Yes, you were correct. I've made some changes which should resolve the indexing issue you mentioned above.
```
$f\left(x\right)=kx+5$ and $g\left(x\right)=-3x+5$ intersects at a point, $(x_1,y_1)$ $(x_1,y_1)$$(x_1,y_1)$$(x_1,y_1)$$(x_1,y_1)$$(x_1,y_1)$$(x_1,y_1)$$(x_1,y_1)$
What is $y_1$
 if k =11.45?
````
This input would fail to render properly on the [360e059](https://github.com/UoaWDCC/nzpmc_frontend/pull/69/commits/360e05989ec09b94fd4f77657744b16a3184697b) but now works properly in [f2879ab](https://github.com/UoaWDCC/nzpmc_frontend/pull/69/commits/f2879ab0e1cd8940df2d476d04e57b7322aeb992)